backend/src/main/java/io/debezium/configserver/model/ConnectorTypeFull.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.configserver.model;
+
+import java.util.List;
+
+public class ConnectorTypeFull extends ConnectorType {
```

Comment:
`Full` feels a bit unspecific, especially as a suffix which is often used to make things more specific (but "full" is not specific) and related to renaming the original `ConnectorType` class to `ConnectorDescriptor` we could maybe keep this name as `ConnectorType`? Or maybe vice versa. I am also fine with a good new name for both.

I could also live with `ConnectorTypeWithProperties`, but I think this class is THE thing what a connector type should be. So the other one must be something different.

Reply:
@rk3rn3r yeah sometimes it's a struggle to come up with appropriate names...
So maybe we could use ConnectorType for 'no properties' and ConnectorDefinition for 'with properties'?

Then the ConnectorIntegrator interfaces would be:

ConnectorType getConnectorType();
ConnectorDefinition getConnectorDefinition();

wdyt?